R/proportional_hazards-glmnet.R

```diff
@@ -326,29 +323,104 @@ multi_predict._coxnet <- function(object,
     }
   }
 
-  if (type == "linear_pred") {
-    pred <- multi_predict_coxnet_linear_pred(
-      object,
-      new_data = new_data,
-      opts = dots,
-      penalty = penalty
-    )
-  } else {
-    pred <- predict(
-      object,
-      new_data = new_data,
-      type = type,
-      ...,
-      penalty = penalty,
-      multi = TRUE
-    )
+  # from predict._coxnet()
+  object$spec$args$penalty <- parsnip::.check_glmnet_penalty_predict(
+    penalty,
+    object,
+    multi = TRUE
+  )
+
+  # from predict.model_fit()
+  if (inherits(object$fit, "try-error")) {
+    rlang::warn("Model fit failed; cannot make predictions.")
+    return(NULL)
+  }
```

Comment:
I've just seen that this is also in the `multi_predict()` generic, will remove after your review!

Reply:
done in [e7e5414](https://github.com/tidymodels/censored/pull/282/commits/e7e54141c36d5affe0dda7f575b04adcc9c77ffc)